Cesium.Compiler/Program.cs

```diff
@@ -54,7 +54,14 @@ static Task<string> Preprocess(TextReader reader)
         var content = await Preprocess(reader);
         var lexer = new CLexer(content);
         var parser = new CParser(lexer);
-        var translationUnit = parser.ParseTranslationUnit().Ok.Value;
+        var translationUnitParseError = parser.ParseTranslationUnit();
+        if (translationUnitParseError.IsError)
+        {
+            var token = (CToken)translationUnitParseError.Error.Got;
```

Comment:
I don't like the nullability warnings in these lines. Let's do something about them?